trakt/users.py

```diff
@@ -480,6 +480,27 @@ def get_stats(self):
         data = yield 'users/{user}/stats'.format(user=slugify(self.username))
         yield data
 
+    @get
+    def get_liked_lists(self, list_type=None, limit=None):
+        """Get items a user likes.
+
+        This will return an array of standard media objects.
+        You can optionally limit the type of results to return.
+
+        list_type possible values are "comments", "lists".
+
+        https://trakt.docs.apiary.io/#reference/users/likes/get-likes
+        """
+        uri = 'users/likes'
+        if list_type is not None:
+            uri += f'/{list_type}'
+
+        if limit is not None:
```

Comment:
you removed the 1000 limit, it was here for a good reason : https://github.com/Taxel/PlexTraktSync/pull/570